setup.cfg

```diff
@@ -61,7 +61,7 @@ formats = zip
 
 [flake8]
 select = B, C, E, F, W, T4, B9
-ignore = E203, E266, E501, W503
+ignore = E203, E266, E501, W503, E741
```

Comment:
hm, I would have rather fixed the offending code, rather than mute the flake8 warning globally

Reply:
I've seen that warning every time you use a single-letter variable like in a list comprehension, so I decided to mute it for now.